### DISCUSSION TITLE
Privacy policy banner

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#5260] [Archive]
Hi, How to present a privacy policy banner when open admin panel? Thanks.

**Comment by benhaynes** [comment]
Probably need to fork the whole codebase for this, or wait for https://github.com/directus/directus/discussions/4590. You could try to use the "Public Note", but not sure if that works for your needs. You could also try adding a Hook that pushes a notification with a link to the page/text.

**Reply by biapar** [reply]
But this redirect user to a other page.

**Reply by benhaynes** [reply]
You can try the options above, if those don't work you can fork the codebase and add a custom overlay on login.
